README.md

```diff
@@ -1,2 +1,87 @@
 # restaurant-js
-A restaurant webpage created using only DOM manipulation with Javascript
+A restaurant webpage created using only DOM manipulation with Javascript and Webpack for learning purposes
+
+## Main Features
+
+* Simple Restaurant Webpage with home, menu and contact information
+
+* Created using only Javascript DOM manipulation  
+
+## Screenshots
+![pic_1](https://user-images.githubusercontent.com/52765379/93238813-509ef980-f758-11ea-9d03-8c43cf8e8843.png)
+
+![pic_2](https://user-images.githubusercontent.com/52765379/93238830-5563ad80-f758-11ea-8f23-2256a8665670.png)
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+- Netlify for deployment
+
+## Live Demo
+
+[Mr. Fresco Live version](https://mrfresco.netlify.app/)
+
+## Getting Started
+
+To running it locally follow these simple steps.
+
+
+### Setup
+
+First build the project with this command:
+
```

Comment:
## Additional changes for setup instructions
Please add instructions on how to install the dependencies here. Thank you
```suggestion
npm install
```

Reply:
You are totally right. Thanks for the suggestion!
Already fixed